tests/vaults/VaultOps.test.ts

```diff
@@ -323,32 +323,86 @@ describe('VaultOps', () => {
   });
   describe('mkdir', () => {
     test('can create directory', async () => {
-      await vaultOps.mkdir(vault, dirName);
+      const response = vaultOps.mkdir(vault, [dirName]);
+      // For await (const _ of response);
+      for await (const data of response) {
+        expect(data.success).toBeTruthy();
+        expect(data.error).toBeUndefined();
+      }
       await expectDirExists(dirName);
     });
     test('can create recursive directory', async () => {
       const dirPath = path.join(dirName, dirName);
-      await vaultOps.mkdir(vault, dirPath, { recursive: true });
+      const response = vaultOps.mkdir(vault, [dirPath], {
+        recursive: true,
+      });
+      // For await (const _ of response);
+      for await (const data of response) {
+        expect(data.success).toBeTruthy();
+        expect(data.error).toBeUndefined();
+      }
```

Comment:
Errant comments here? If they're not needed then you should remove them.

Reply:
Ah, oops.